framework/operator_test.go

```diff
@@ -1,4 +1,4 @@
-package operator
+package framework
```

Comment:
Shouldn't also operator.go and operator_test.go be renamed and all `Test_Oerator` functions?
Is there also any `Operator` struct that should be renamed?